src/layout/styles.js

```diff
@@ -29,8 +29,7 @@ const Body = styled.div`
 const BodyGeneral = styled.div`
   position: relative;
   flex: 1;
-  padding: 5rem 1.5rem 3rem;
-  padding: 5rem 1.5rem 3rem;
+  padding: ${({ noPadding }) => (noPadding ? '0' : '5rem 1.5rem 3rem')};
```

Comment:
lets rather not do this, there will likely be a header, and some content above and below this map.